src/Tools/setup-snapshot.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+PCL_NAME=PortableReferenceAssemblies-2014-04-14
+
+if [ "$EUID" -ne 0 ]; then
+    echo "Error: This script must be run as root"
+    exit 1
+fi
+
+apt-get -y install mono-snapshot-latest
+
```

Comment:
Do we always want latest mono? 
This way we would catch breaking changes in mono early, but is this intentional?